### ISSUE TITLE
earnings not working properly

### ISSUE THREAD

**Issue by Timobaaij** [ranaroussi/yfinance#945]
Just tried to call earnings dataframe for ticker: JAV2.F

The data is clearly on the Yahoo Finance website but the function returns an empty DF with the wrong columns. 

![image](https://user-images.githubusercontent.com/39728252/151656697-cdd8096d-63af-4915-a7e1-0c5ce33f08eb.png)

![image](https://user-images.githubusercontent.com/39728252/151656716-89e8f423-a070-4cb9-97f8-0d4f61b88c5a.png)

Can you please tell me how to fix this?
Anyone else experiencing this problem?

**Comment by Timobaaij** [comment]
BTW it is not a stand alone problem - there are more tickers where I experience this.

**Comment by ProgrammingDao** [comment]
I have had a quick look and it seems that utils.get_json('https://finance.yahoo.com/quote/JAV2.F/financials') queries yahoo finance and gets an html without any earning data in:
 _json.loads(json_str)['context']['dispatcher']['stores']['QuoteSummaryStore'] (utils.py line 108)

there might be more information in:
 _json.loads(json_str)['context']['dispatcher']['stores']['QuoteTimeSeriesStore']['timeSeries'] (utils.py line 112) but not exactly what you're looking for.
 
 To sum up, the issue comes from the data fetched from yahoo instead of yfinance package.

**Comment by Timobaaij** [comment]
Yes, it is probably not there... the earnings function is not working but you can find the data in de financials function under "total revenue". 

There are many issues reported for this I found
